compiler/rustc_macros/src/symbols.rs

```diff
@@ -315,10 +315,11 @@ fn symbols_with_errors(input: TokenStream) -> (TokenStream, Vec<syn::Error>) {
         }
 
         impl Interner {
-            pub(crate) fn fresh() -> Self {
-                Interner::prefill(&[
-                    #prefill_stream
-                ])
+            pub(crate) fn fresh(extra: &[&'static str]) -> Self {
```

Comment:
I would recommend keeping `fresh()` but naming a new method for this. Perhaps `new_with_extra_preinterned_symbols`?

Reply:
Renamed it to `with_extra_symbols` and added some docs

There's only one callsite so the current `fresh` would be unused